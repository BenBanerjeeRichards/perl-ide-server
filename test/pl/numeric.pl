2.3;
34;
0xfe34e;
23e34;
-23e56;
23e-34;
-34e-43;
0b101001;
34.03;
34_34_34;
34_34_34e34_34;
_45;
