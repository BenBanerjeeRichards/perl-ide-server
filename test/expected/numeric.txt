NumericLiteral(2.3)
;
NumericLiteral(34)
;
NumericLiteral(0xfe34e)
;
NumericLiteral(23e34)
;
NumericLiteral(-23e56)
;
NumericLiteral(23e-34)
;
NumericLiteral(-34e-43)
;
NumericLiteral(0b101001)
;
NumericLiteral(34.03)
;
NumericLiteral(34_34_34)
;
NumericLiteral(34_34_34e34_34)
;
NumericLiteral(34.34_23_10)
;
VersionLiteral(v54.32)
;
VersionLiteral(34.34.3)
;
VersionLiteral(v4.5)
;
Name(_45)
;
EndOfInput
